src/main/java/org/irods/serialize/ModifyPermissionsOperations.java

```diff
@@ -1,20 +1,33 @@
 package org.irods.serialize;
 
-
 import com.fasterxml.jackson.annotation.JsonProperty;
 import org.irods.util.Permission;
 
 /**
- * For the modify_permissions() method.
- * Represents the JSON object that is passed into the operations parameter
+ * Represents the JSON object for the {@code modifyPermissions()} method used in the collections and data-objects
+ * endpoint. This class encapsulates the parameters for modifying permissions.
  */
 public class ModifyPermissionsOperations {
-    @JsonProperty("")
+    @JsonProperty("entity_name")
     private String entity_name;
 
     @JsonProperty("acl")
     private String acl;
 
+    /**
+     * Constructs a new instance of {@code ModifyPermissionsOperations}.
+     *
+     * @param entity_name The name of the user or group.
+     * @param perm The permission level to set for the entity. Must be one of the following values:
+     *             <ul>
+     *                  <li>{@code Permission.NULL}</li>
+     *                  <li>{@code Permission.READ}</li>
+     *                  <li>{@code Permission.WRITE}</li>
+     *                  <li>{@code Permission.OWN}</li>
```

Comment:
is this true? or did we get to all 10?

Reply:
Get to all 10 of the permissions? I added all the permissions from docs.irods.org into the Permission enum

Reply:
@korydraughn do we want to list all 10 permissions here?   seems... correct.

Reply:
No. The atomic ACLs API does not support all permission levels yet. We'll need to open an issue for that if we don't have one already.

Reply:
The `setPermission` does take all permission levels, the `modifyPermission` does not. They both are using the `Permission` enum that contains all the permission levels, but if one of those permissions are not one of the expected permission levels for `modifyPermission`, an error is thrown.

Reply:
Correct. Marking as resolved.